sdk/state/payment.go

```diff
@@ -54,6 +54,13 @@ func (c *Channel) ProposePayment(amount int64) (CloseAgreement, error) {
 	if amount <= 0 {
 		return CloseAgreement{}, errors.New("payment amount must be greater than 0")
 	}
+
+	// If a coordinated close has been accepted already, error.
```

Comment:
❗ We should also check if we have proposed a coordinated close by checking if the `latestUnauthorizedCloseAgreement` has zeroes too. If we've proposed a coordinated close we shouldn't ever propose or sign any new close agreements, even if we never get a response back from the other participant. The reason being that they could sign and submit that close at anytime, even if we've both agreed to new close agreements after it.

Reply:
👍 added here: https://github.com/stellar/experimental-payment-channels/pull/197/commits/e342078733285002ecf2f183ae7586a23744d240